src/map/packets_struct.h

```diff
@@ -3533,6 +3533,49 @@ struct PACKET_ZC_HAT_EFFECT {
 DEFINE_PACKET_HEADER(ZC_HAT_EFFECT, 0x0a3b);
 #endif
 
+#if PACKETVER_MAIN_NUM >= 20190508 || PACKETVER_RE_NUM >= 20190508 || PACKETVER_ZERO_NUM >= 20190502
```

Comment:
for packet 0xb27 starting dates:
2019-07-17cRagexeRE
2019-07-31bRagexe
zero not supported yet